plugins/modules/foreman_bookmark.py

```diff
@@ -115,15 +119,9 @@ def main():
     )
 
     entity_dict = module.clean_params()
+    module.search = 'name="{0}",controller="{1}"'.format(entity_dict['name'], entity_dict['controller'])
```

Comment:
I think this feels like an odd pattern. Setting properties and then using that later feels hard to follow. Not sure what would be a better design.

Reply:
Yes, moved to method param.